src/examples/rest/src/backend/schema/person.ts

```diff
@@ -17,13 +17,16 @@ import { Vehicle } from './vehicle';
 	}),
 })
 export class Person {
-	@Field(() => ID, { primaryKeyField: true })
+	@Field(() => ID, {
+		primaryKeyField: true,
+		adminUIOptions: { filterType: AdminUIFilterType.DROP_DOWN_TEXT },
+	})
 	url!: string;
 
-	@Field(() => String)
+	@Field(() => String, { adminUIOptions: { filterType: AdminUIFilterType.TEXT } })
```

Comment:
Since this is the default let's just do:
```suggestion
	@Field(() => String)
```